primitives/runtime/src/traits.rs

```diff
@@ -134,28 +134,6 @@ impl<
 	}
 }
 
-/// An error type that indicates that the origin is invalid.
```

Comment:
I think the trait should stay here and we introduce the `BadOrigin` type directly here.

We already have some inherent errors that are declared by `decl_error!`. I would add a new one `BadOrigin` that implements `From<BadOrigin>` that is defined in this file here.

Reply:
I have some upcoming changes in my mind for the error stuff, that would conflict in the way it is implemented currently.